lib/HTTP/BrowserDetect.pm

```diff
@@ -394,68 +395,96 @@ sub user_agent {
 # Private method -- Set up the basics (browser and misc attributes)
 # for a new user-agent string
 sub _init_core {
```

Comment:
I don't think we need to preserve this functionality here.  I'm happy to see it go and force everyone to use ->new()